src/components/Popup.component.tsx

```diff
@@ -98,7 +98,7 @@ export default function Popup({
               >
                 <Dialog.Panel className="w-full max-w-sm transform overflow-hidden rounded-2xl inv-background text-left align-middle shadow-xl transition-all">
                   <div className="flex flex-col items-center gap-2">
-                    <Slot item={recipes[solution].output} clickable={false} />
+                    <CraftingTable solved={true} />
```

Comment:
you will need to pass the args for active and table num here, or make them optional in CraftingTable.component because this fails to compile.